cont.c

```diff
@@ -3264,12 +3266,18 @@ rb_fiber_m_raise(int argc, VALUE *argv, VALUE self)
  *  call-seq:
  *     fiber.kill -> nil
  *
- *  Terminates +fiber+ by raising an uncatchable exception, returning
- *  the terminated Fiber.
+ *  Terminates +fiber+ by raising an uncatchable exception.
```

Comment:
Just because it's uncatchable, does it mean it can be seen in `$!`? I think we should be careful about this wording, as in practice, no exception should be allocated, at least in CRuby.

Reply:
I think it's the closest thing we can say, things like throw tags are CRuby impl details.
Conceptually it behaves like an uncatchable exception.
I think "uncatchable" kinda implies "will not set $!", because that would sort of capture it and also $! is only ever set to an instance of Ruby Exception.